test/unit/org/apache/cassandra/cql3/validation/entities/SecondaryIndexTest.java

```diff
@@ -1575,7 +1575,7 @@ private void testIndexOnPartitionKeyInsertExpiringColumn(boolean flushBeforeUpda
             flush();
 
         execute("UPDATE %s USING TTL 1 SET b = 10 WHERE k1 = 1 AND k2 = 2");
-        Thread.sleep(1000);
+        Thread.sleep(1500);
```

Comment:
Why this change?

Reply:
If you insert with TTL=1 don't ask for trouble (flakies) and sleep a bit more than 1s. Iirc this failed once on me. In fact I just did the same on the next test which also inserts with TTL=1

Reply:
It would be nice to not rely on fudge factors if we can.

Reply:
What would be a good alternative?

Reply:
Tests should be deterministic, and it's up to authors to make that happen. In this case, probably the easiest thing to do is to execute both with a provided timestamp and nowInSeconds, for which there are existing methods in QueryProcessor.

Reply:
Then I'll say stabilizing other tests is outside the scope and I don't want to open that can of worms here. I can either A. Revert the change B. Open a ticket or A+B, wdyt?

Reply:
Reverted in latest commit in case it was a side effect of one of my changes at the time during dev. I'll open a ticket if it surfaces again.